PocketApp.swift

```diff
@@ -12,6 +12,10 @@ struct PocketApp: App {
     var body: some Scene {
         WindowGroup {
             RootView(model: RootViewModel())
+                .onOpenURL { url in
+                    guard url.scheme == "widget-deeplink" else { return }
```

Comment:
this works, but we probably want to leave `PocketApp` as agnostic as possible. An alternative could be to just call a more generic method in the appDelegate (for example `handle(url: URL)` and then let this one handle the logic? This way, we can handle different types of urls and the main does not need to know anything about it

Reply:
will update!